app/grandchallenge/challenges/management/commands/set_challenge_request_handling_preference.py

```diff
@@ -0,0 +1,18 @@
+from django.core.management import BaseCommand
+
+from grandchallenge.challenges.models import Challenge
+from grandchallenge.core.utils.access_request_utils import (
+    AccessRequestHandlingOptions,
+)
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        challenges = Challenge.objects.filter(require_participant_review=False)
+        for challenge in challenges:
+            challenge.access_request_handling = (
+                AccessRequestHandlingOptions.ACCEPT_ALL
+            )
+
+        Challenge.objects.bulk_update(challenges, ["access_request_handling"])

```

Comment:
This doesn't matter as this is a throwaway command but just FYI you can do this in one swoop:

```
Challenge.objects.filter(require_participant_review=False).update(access_request_handling=AccessRequestHandlingOptions.ACCEPT_ALL)
```